docs/docs/dev_docs/getting_started/sandbox.md

```diff
@@ -60,18 +62,29 @@ With the help of Aztec.js you will be able to:
 
 ## I have the Sandbox running, show me how to use it!
 
-Let's have a complete walkthrough from start to finish. I'm using WSL2 Ubuntu under Windows but the following should work under regular Linux or MacOS. We will deploy and use a private token contract on our Sandbox. Writing the contract itself is out of scope for this tutorial, we will use the Private Token Contract supplied as one of the example contracts.
+We will deploy a private token contract, and send tokens privately, using the Sandbox.
+
+> Writing the contract itself is out of scope for this tutorial, we will use a Private Token Contract which as been pre-supplied as an example. See [here](LINK) for more information on how to write contracts for Aztec.
```

Comment:
Whoops, sorry, my code suggestion was lazy and didn't actually provide you with a valid link. This'll need to be a valid link for the docs to actually build, via `yarn build`.